flake-modules/tests.nix

```
@@ -39,6 +40,17 @@
           nixvim = self;
         })
         .activationPackage;
+
+      nixos =
```

```suggestion
      nixos-module =
```

And similarly, `home-manager-module` above.
I feel like it's more explicit.

---

Done. Also disabled the nixos checks for darwin